packages/api-react/src/hooks/useGetNFTWallets.ts

```diff
@@ -1,4 +1,5 @@
-import { Wallet, WalletType } from '@chia-network/api';
+import type { Wallet } from '@chia-network/api';
+import WalletType from '@chia-network/api';
```

Comment:
`WalletType` is being imported as undefined here. I tested the following and was able to get the NFT wallets to return the correct listing.

```suggestion
import { WalletType } from '@chia-network/api';
```